db/seeds.rb

```diff
@@ -1,22 +1,26 @@
 puts "Seeding Employees..."
-1.upto(5) do |i|
-    Employee.create!(email: "emp#{i}@rera.com", password: 'zaq123', number_of_available_kudos: Faker::Number.between(from: 1, to: 10))
+1.upto(10) do |i|
+    Employee.create!(email: "emp#{i}@rera.com", password: 'zaq123', number_of_available_kudos: rand(1..10))
 end
-puts "Done."
+puts "Done. Created #{Employee.count} Employees"
 
 puts "Seeding Admins..."
     AdminUser.create!(email: "admin@rera.com", password: 'zaq123')
 puts "Done."
 
-puts "Seeding Kudos..."
-5.times do
-    Kudo.create!(title: Faker::Superhero.name, content: Faker::TvShows::Friends.quote, giver_id: Faker::Number.between(from: 1, to: 5), receiver_id: Faker::Number.between(from: 1, to: 5))
-end
-puts "Done."
-
 puts "Seeding Company Values..."
     CompanyValue.create!(title: 'Honesty')
     CompanyValue.create!(title: 'Ownership')
     CompanyValue.create!(title: 'Accountability')
     CompanyValue.create!(title: 'Passion')
-puts "Done."
+puts "Done. Created #{CompanyValue.count} Company Values"
+
+puts "Seeding Kudos..."
+10.times do
+    Kudo.create!(   title: Faker::Superhero.name, 
+                    content: Faker::TvShows::Friends.quote, 
+                    giver_id: rand(1..10), 
```

Comment:
Since later on it may be hard to track how many Employees, Rewards, or Company Values you have consider using (same thing applies to other relations): 
```suggestion
                    giver: Employee.sample
```

Reply:
Corrected ;)